tools/depends/target/nettle/Makefile

```diff
@@ -22,6 +22,7 @@ $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 	rm -rf $(PLATFORM)/*; mkdir -p $(PLATFORM)
 	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
 	cd $(PLATFORM); patch -p1 -i ../01-disable_testsuite.patch
+	cd $(PLATFORM); patch -p1 -i ../02-conftest_exit.patch
```

Comment:
You also need to add this script to the DEPS at the top of this file

Reply:
Done